packages/grafana-ui/src/components/Layout/Space.mdx

```diff
@@ -0,0 +1,22 @@
+import { Meta, ArgTypes } from '@storybook/blocks';
+import { Space } from './Space';
+
+<Meta title="MDX|Space" component={Space} />
+
+# Space
+
+The `Space` component is a wrapper around the `Box` component, used to add space between elements. Horizontal space is added using the `h` prop, while vertical space is added using the `v` prop.
```

Comment:
> is a wrapper around the `Box` component

I don't think you need to say this. This is an unimportant implementation detail.